helm/chaos-mesh/values.yaml

```diff
@@ -43,7 +43,7 @@ enableProfiling: true
 # enableCtrlServer is a flag to enable ctrlserver in controller-manager and chaos-daemon.
 enableCtrlServer: true
 
-registry: ""
+registry: "ghcr.io"
```

Comment:
Could we add a global `tag` here, and if user specifies the component `tag`, it will be used in preference, if not, the global `tag` will be used.

Reply:
OK!

Reply:
> Could we add a global `tag` here, and if user specifies the component `tag`, it will be used in preference, if not, the global `tag` will be used.

Do we need the same thing on `registry`?

Reply:
I think it's a great idea :D

Reply:
> I think it's a great idea :D

Updated! PTAL